src/brouwers/kitreviews/models.py

```diff
@@ -21,28 +24,28 @@ class KitReview(models.Model):
     raw_text = models.TextField(
         _(u'review'),
         help_text=_('This is your review. You can use BBCode here.')
-        )
+    )
     html = models.TextField(blank=True, help_text=u'raw_text with BBCode rendered as html')
-    positive_points = models.TextField(_('positive points'), blank=True)
-    negative_points = models.TextField(_('negative points'), blank=True)
-    rating = models.PositiveSmallIntegerField(_(u'rating'), default=DEFAULT_RATING)
+    prop = models.ManyToManyField('KitReviewProperty', blank=True, related_name='reviews')
+    rating = models.PositiveSmallIntegerField(_(u'rating'), default=DEFAULT_RATING,
```

Comment:
The JavaScript slider will then probably set a default value :(
On Apr 4, 2016 6:02 PM, "Alex Khomenko" <notifications@github.com> wrote:

> In src/brouwers/kitreviews/models.py
> <https://github.com/modelbrouwers/modelbrouwers/pull/47#discussion_r58400581>
> :
>
> >      html = models.TextField(blank=True, help_text=u'raw_text with BBCode rendered as html')
> > -    positive_points = models.TextField(_('positive points'), blank=True)
> > -    negative_points = models.TextField(_('negative points'), blank=True)
> > -    rating = models.PositiveSmallIntegerField(_(u'rating'), default=DEFAULT_RATING)
> > +    prop = models.ManyToManyField('KitReviewProperty', blank=True, related_name='reviews')
> > +    rating = models.PositiveSmallIntegerField(_(u'rating'), default=DEFAULT_RATING,
>
> Good point. What if we don't have a default value for general rating then
> and make it required?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/modelbrouwers/modelbrouwers/pull/47/files/c5219b1fa2e9b039ecf102ec03e4ec19127e646c#r58400581>
>